### DISCUSSION TITLE
User define Task for Mongo DB CRUD Operation

### DISCUSSION THREAD

**Discussion by vishalduke** [Netflix/conductor#3249] [Q&A]
can we have a User Defined Task type that can perform CRUD operations with Mogo DB?
the reason we are asking, as we are seeking help or guidance where 
1. **We will define USER Define Task**
2. **Created task will appear or can be used by Conductor at design time using Conductor UI**

**Comment by apanicker-nflx** [comment]
You can create your task within your own service which performs CRUD operations on MongoDB. This task execution can be housed in a worker and the task can be added to a workflow.
The assumption here is that you are using MongoDB for some business use case and not as a persistence store for Conductor.
